test/integration-test-main.sh

```diff
@@ -63,9 +63,6 @@ function rm_test_dir {
 
 function test_append_file {
     echo "Testing append to file ..."
-    # Open and close file outside of loop to avoid eventual object consistency
-    # issues
-    exec 3<> ${TEST_TEXT_FILE}  # open
     # Write a small test file
```

Comment:
@andrewgaul suggestion was much simpler but I missed removing these lines when I switched to it.  It doesn't break anything but is confusing.